submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/processors/ProjectProcessor.kt

```diff
@@ -0,0 +1,11 @@
+package ac.uk.ebi.biostd.submission.processors
+
+import ebi.ac.uk.model.ExtendedSubmission
+import ebi.ac.uk.model.extensions.addAccessTag
+import ebi.ac.uk.persistence.PersistenceContext
+
+class ProjectProcessor : SubmissionProcessor {
```

Comment:
Is this only applied to the project? as it implements SubmissionProcessor I would expect is applied to every submission and it does look correct them

Reply:
Well, the project is a submission. In this case, this is a processor for the projects which follows the same validation/processing approach.

Maybe we could rename the interface as `SubmitterProcessor` and `SubmitterValidator`?